README.markdown

```diff
@@ -423,18 +424,18 @@ It is also extremely appreciated to use comments to separate the [Class Attribut
 
 **Preferred:**
 ```swift
-class MyViewController                      : UIViewController {
+class MyViewController                           : UIViewController {
 
     // MARK: - Outlets
 
-    @IBOutlet var lettersGameButton         : UIButton?
-    @IBOutlet var headBodyLegsGameButton    : UIButton?
-    @IBOutlet var colorMatcherGameButton    : UIButton?
+    @IBOutlet weak var lettersGameButton         : UIButton?
+    @IBOutlet weak var headBodyLegsGameButton    : UIButton?
+    @IBOutlet weak var colorMatcherGameButton    : UIButton?
 
```

Comment:
IBOutlets should always be weak to avoid retain cycles.
(one more reason to drag Outlet from IB: the generated code is fine)